README.md

```diff
@@ -129,6 +129,6 @@ You can see how we do it in our own repository for SonarLint Connected mode (and
 
 ## License
 
-Copyright 2014-2021 SonarSource.
+Copyright 2014-2022 SonarSource.

```

Comment:
where could we automate this?

should we add it to https://github.com/SonarSource/languages-experimental-tooling/tree/master/personal/pavel-mikula/DotNetLicenseHeaders ? 

(and maybe improve it to work for AutoScan and S4NET , too?)

Reply:
because https://github.com/SonarSource/sonar-scanner-msbuild/pull/1219/files

Reply:
Yes, it should be caught by the `DumpUnexpected()` part. It is not, because the tool only cares about `analyzers` subdirectory.  There should be no harm in inspecting it from the root. 

I'll do that. Thanks for the ping.

Reply:
Done

Reply:
LOL, the tool is pretty powerfull. I should have run it before approving 😅 

https://github.com/SonarSource/sonar-dotnet/blob/5b7fca1014b5bf867340611ffb95367d0e522c82/NOTICE.txt#L2

Reply:
I will create a PR. There are many other files to fix